Tests/SwiftParserTest/Declarations.swift

```diff
@@ -804,7 +804,7 @@ final class DeclarationTests: XCTestCase {
 
   func testDontRecoverFromDeclKeyword() {
     AssertParse(
-      "func foo(first second 1️⃣third 3️⃣struct4️⃣: Int) {}",
+      "func foo(first second 1️⃣third 2️⃣struct3️⃣: Int4️⃣) {}",
```

Comment:
This was previously parsing just `struct` with all unexpected, but now (after also checking for start of line) parses the remained as a `struct` instead.